### ISSUE TITLE
New resource: aws_media_package_channel

### ISSUE THREAD

**Issue by sjauld** [terraform-providers/terraform-provider-aws#6930]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I have just started dabbling with the AWS Elemental stuff and it would be great to have their resources in terraform. I'll start with this one and go from there. 

### New or Affected Resource(s)

* aws_media_package_channel

### Potential Terraform Configuration

TBC

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* https://docs.aws.amazon.com/mediapackage/latest/ug/channels.htm
* https://docs.aws.amazon.com/sdk-for-go/api/service/mediapackage/

**Comment by bflad** [comment]
The new `aws_media_package_channel` resource was merged and will release with version 1.55.0 of the AWS provider, likely in the next two days. 👍

**Comment by bflad** [comment]
This has been released in [version 1.55.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1550-january-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
